---a/net/minecraft/command/server/CommandBroadcast.java
+++b/net/minecraft/command/server/CommandBroadcast.java
@@ -1,6 +1,7 @@
 package net.minecraft.command.server;
 
 import java.util.List;
+
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommandSender;
@@ -10,41 +10,33 @@
 import net.minecraft.util.ChatComponentTranslation;
 import net.minecraft.util.IChatComponent;
 
-public class CommandBroadcast extends CommandBase
-{
+public class CommandBroadcast extends CommandBase {
+
     private static final String __OBFID = "CL_00000191";
 
-    public String getName()
-    {
+    public String getName() {
         return "say";
     }
 
-    public int getRequiredPermissionLevel()
-    {
+    public int getRequiredPermissionLevel() {
         return 1;
     }
 
-    public String getCommandUsage(ICommandSender sender)
-    {
+    public String getCommandUsage(ICommandSender sender) {
         return "commands.say.usage";
     }
 
-    public void execute(ICommandSender sender, String[] args) throws CommandException
-    {
-        if (args.length > 0 && args[0].length() > 0)
-        {
+    public void execute(ICommandSender sender, String[] args) throws CommandException {
+        if (args.length > 0 && args[0].length() > 0) {
             IChatComponent ichatcomponent = getChatComponentFromNthArg(sender, args, 0, true);
-            MinecraftServer.getServer().getConfigurationManager().sendChatMsg(new ChatComponentTranslation("chat.type.announcement", new Object[] {sender.getDisplayName(), ichatcomponent}));
-        }
-        else
-        {
+            MinecraftServer.getServer().getConfigurationManager().sendChatMsg(new ChatComponentTranslation("chat.type.announcement", new Object[]{sender.getDisplayName(), ichatcomponent}));
+        } else {
             throw new WrongUsageException("commands.say.usage", new Object[0]);
         }
     }
 
-    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos)
-    {
+    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
         return args.length >= 1 ? getListOfStringsMatchingLastWord(args, MinecraftServer.getServer().getAllUsernames()) : null;
     }
-}
+}
